### PR TITLE
Preview action improvements 2

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_ui/issues/168

* [x] factorise a bit more the view/preview mako templates? (but since helper functions are used, the common code is not that big)

### CODE REVIEW

**Comment by asaunier** [comment]
I don't have time to make more improvements in this PR (see the list in https://github.com/c2corg/v6_ui/issues/168#issuecomment-245893257) and would like to merge it as is before more conflicts come with it (I have just rebased the PR and fixed conflicts).

Please review.

**Comment by asaunier** [comment]
Updated to get the Travis fix

File: c2corg_ui/templates/helpers/view.html [hunk 1]
```diff
@@ -272,6 +272,36 @@ <h3 class="heading show-phone outings" ng-click="mainCtrl.animateHeaderIcon($eve
...
+          % if locale.get('summary'):
+            <summary class="document-summary">
+              <label translate>summary</label><br>
+              ${show_attr(locale, 'summary')}
+            </summary>
+          % endif
+
+          % if locale.get('description'):
+            ${show_attr(locale, 'description')}
+          % else :
```
Comment (tsauerwein) [inline c2corg_ui/templates/helpers/view.html]: minor: blank space after else

Comment (ginold) [inline c2corg_ui/templates/helpers/view.html]: yeah you too, finally a blank space !

Comment (asaunier) [inline c2corg_ui/templates/helpers/view.html]: @ginold sorry for breaking the myth :P

**Comment by asaunier** [comment]
@tsauerwein thanks for reviewing. I have fixed the extra blank space and another typo (class "test-center" instead of "text-center" for the images).

**Pull request merged**